tests/testthat/_snaps/tbl_ae_rates_by_grade.md

```diff
@@ -0,0 +1,581 @@
+# tbl_ae_rates_by_grade() works
```

Comment:
Can you make this snapshot smaller? Just enough rows to show it's working

Reply:
Updated